app/models/controlled_work_document_content.rb

```diff
@@ -13,13 +13,16 @@
 class ControlledWorkDocumentContent < Check
   include ActionView::Helpers::NumberHelper
 
+  attr_reader :form_type
+
   def from_cfe_payload(path)
     path_parts = path.split(".").map { _1.to_i.to_s == _1 ? _1.to_i : _1 }
     value = session_data.dig("api_response", *path_parts)
     format(value)
   end
 
-  def from_attribute(attribute)
+  def from_attribute(attribute, form_type = nil)
+    @form_type = form_type
```

Comment:
Be wary of setting a permanent attribute on a model from a temporary method. In future, when new methods are added, we may not remember that depending on whether `from_attribute` had been called before or after it, `@form_type` may or may not be set and maybe this will affect how that new method operates.